scripts/lib/setup_venv.py

```diff
@@ -320,6 +320,21 @@ def do_setup_virtualenv(venv_path, requirements_file, virtualenv_args):
         print("Configuring pip to use custom CA certificates...")
         add_cert_to_pipconf()
 
+    # CentOS-specific
+    # Install pycurl with custom flag due to this error when installing
+    # via pip:
+    # __main__.ConfigurationError: Curl is configured to use SSL, but
+    # we have not been able to determine which SSL backend it is using.
+    # Please see PycURL documentation for how to specify the SSL
+    # backend manually.
+    # See https://github.com/pycurl/pycurl/issues/526
+    # The fix exists on master, but not yet on release
+    if os.path.exists("/etc/redhat-release"):
+        pycurl_env = os.environ.copy()
+        pycurl_env["PYCURL_SSL_LIBRARY"] = "nss"
+        run("pip install pycurl==7.43.0.2 --compile --no-cache-dir".split(' '),
```

Comment:
Is there any particular reason to not use this `'string command'.split(' ')` but instead with an explicit list?

Reply:
Yes, it's bad style :).

Reply:
Why is it safer with a list? Wouldn't the string be more readable when there are no variable formatting required?

Reply:
Because it's the correct way to write things when you have variables involved.